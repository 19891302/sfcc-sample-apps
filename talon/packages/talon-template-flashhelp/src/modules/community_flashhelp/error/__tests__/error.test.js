import { createElement } from 'lwc';
import ErrorCmp from 'community_flashhelp/error';

describe('community_flashhelp/error', () => {
    beforeEach(() => {
        while (document.body.firstChild) {
            document.body.removeChild(document.body.firstChild);
        }
    });

    it('renders correctly', () => {
        const element = createElement('community_flashhelp-error', { is: ErrorCmp });
        document.body.appendChild(element);

        return Promise.resolve().then(() => {
            expect(element).toMatchSnapshot();
        });
    });
});
