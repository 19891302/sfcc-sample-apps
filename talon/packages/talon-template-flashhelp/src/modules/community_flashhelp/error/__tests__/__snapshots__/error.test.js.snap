// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`community_flashhelp/error renders correctly 1`] = `
<community_flashhelp-error>
  #shadow-root(open)
    Error
</community_flashhelp-error>
`;
